megamek/src/megamek/server/Server.java

```diff
@@ -4277,25 +4138,24 @@ private void applyDropshipProximityDamage(Coords centralPos,
         // ok now I need to look at the damage rings - start at 2 and go to 7
         for (int i = 2; i < 8; i++) {
             int damageDice = (8 - i) * 2;
-            ArrayList<Coords> ring = Compute.coordsAtRange(centralPos, i);
+            List<Coords> ring = Compute.coordsAtRange(centralPos, i);
             for (Coords pos : ring) {
-                if (rearArc
-                    && !Compute.isInArc(centralPos, facing, pos,
-                                        Compute.ARC_AFT)) {
+                if (rearArc && !Compute.isInArc(centralPos, facing, pos, Compute.ARC_AFT)) {
                     continue;
                 }
 
-                alreadyHit = artilleryDamageHex(pos, centralPos, damageDice,
-                                                null, killer.getId(), killer, null, false, 0,
-                                                vPhaseReport, false, alreadyHit, true);
+                alreadyHit = artilleryDamageHex(pos, centralPos, damageDice, null, killer.getId(),
+                        killer, null, false, 0, vPhaseReport, false,
+                        alreadyHit, true);
 
+                // TODO : Windchild can this be removed?
```

Comment:
Can this commented out code be removed?

Reply:
I'm not sure what was going on here, but it's been like that for over six years. Let's get rid of it.